docs/submission/submitting-clinical-data.md

```diff
@@ -63,3 +63,48 @@ Uploaded data must be checked for accuracy against the data in the current submi
 1. If your data does not require approval by DCC, it will be submitted automatically.
 
 Once submitted, you will see the clinical data reflected on your [Program Dashboard](/docs/submission/submitted-data).
+
+## Submitting data in Surgery file
+
+#### How to indicate a donor who had Mastectomy surgery:
+
+- Submit 'treatment_type' = 'Surgery' and assign a unique submitter_treatment_id ('Tr-1' in this example):
+
+Treatment File
+![Treatment_Example1](/assets/submission/treatment_ex1.png)
+
+- Submit 'surgery_type' = 'Mastectomy' for the same submitter_treatment_id submitted in the Treatment file ('Tr-1), and indicate relevant details about surgery:
+
+Surgery File
+![Surgery Example1](/assets/submission/surgery_ex1.png)
+
+#### How to indicate a donor who had two surgeries at two different times.
+
+- Submit 'treatment_type' = 'Surgery' and assign a unique submitter_treatment_id to each surgery treatment event ('Tr-1' and 'Tr-2' in this example). Indicate treatment_start_interval and treatment_duration for each surgery treatment event.
```

Comment:
Are there any restrictions or issues that arise if there are overlapping treatment intervals?  If so should we make the submitter aware of them here?

Reply:
It's possible a donor could have two surgeries done on the same day (ie. `treatment_start_interval` would be the same for both `Tr-1` and `Tr-2` in this case). This won't result in any errors because it is a valid use case.

Reply:
Thanks for clarification, no issue